src/endpoints/event.endpoint.js

```diff
@@ -165,11 +166,11 @@ const deleteEvent = (req, res) => {
     }
 };
 
-router.get('/', getAllEvents);
-router.get('/:id', getEventById);
-router.get('/title/:title', getEventByTitle);
-router.post('/', postEvent);
-router.put('/:id', updateEvent);
-router.delete('/:id', deleteEvent);
+router.get('/', passport.authenticate('basic', { session: false }), getAllEvents);
```

Comment:
You could use something like this so you don't need to add the authentication middleware to every single route:

`app.all('*', passport.authenticate('basic', { session: false }));`